open-company/open-company-web
# Comment reactions.  Add a single thumbs up emoji to allow the user to "agree" to the comment.

Display and allow a user to react to a comment.

https://trello.com/c/lD7AaDcf/836-comment-reactions-agree-thumbs-up

Depends on https://github.com/open-company/open-company-interaction/pull/10


To test
- Make sure you are using the above `comment-reactions` branch for the interaction service.
- Login and add a comment to an entry.
- [x]  Is there a default 👍  emoji next to the comment that is grayed out?
- [x]  Does the gray get removed when hovering over the 👍 ?
- Click the thumbs up.
- [x]  The counter should increase by 1 and the gray color should be removed.
- Click the thumbs up.
- [x]  The counter should go to zero and the 👍  should be gray.
- Click the thumbs up again So that the second test will work.

- Login as a different user.
- Open the entry to see the comment you created with the other user.
- [x] Does it have a count of one with a colored 👍 ?
- Click the 👍 
- [ ] Does the count go to 2?
- Click the 👍 
- [ ] Does the count go to 1?
